### DISCUSSION TITLE
Direct SDCard support for ESP8266?

### DISCUSSION THREAD

**Discussion by rossiniscarface** [luc-github/ESP3D#597] [Questions]
Hi, 
as i am looking into use ESP3D for my CNC3018, i would like to know if it is possible to hook up a sd-card to the esp8266 like mentioned here [`How to use SD card with esp8266, esp32 and Arduino`](https://www.mischianti.org/2019/12/15/how-to-use-sd-card-with-esp8266-and-arduino/) and then start send a gcode-file to the cnc?
As mostly of the cnc3018 boards don't have a sd-card slot, or the "offline"-Controllers using the serial connection, this would be a nice addition.

**Comment by luc-github** [comment]
the 3.0 does support direct sd card connection but does not not stream gcode yet https://github.com/luc-github/ESP3D/issues/243
